### PR TITLE
added USB001.001 and USB002.001

### PR DESCRIPTION
**Pull request by filipleple**

### CODE REVIEW

File: lib/usb-hid-msc-lib.robot [hunk 1]
```diff
@@ -0,0 +1,35 @@
+*** Keywords ***
+Mount Bootable ISO
+    [Documentation]    Mounts a bootable ISO as flash USB. Currently
+    ...    only the Qubes OS ISO seems to work for the platform.
+    Mount Image    ${PIKVM_IP}    Qubes-R4.2.0-rc1-x86_64.iso
```
Comment (macpijan) [inline lib/usb-hid-msc-lib.robot:5]: This assumes we already have such image on PiKVM. Are we sure we do on each of them? Wouldn't it be easier to upload some smaller image (maybe DTS?) and mount it? You will need such keyword anyway for DTS tests, most likely.

File: lib/usb-hid-msc-lib.robot [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Navigate Through Some Menus
+    [Documentation]    Attempts to navigate And Enter some menus
+    ...    using the USB keyboard to verify that it works.
+    Press Key N Times    1    ${ENTER}
+    Read From Terminal Until    Standard Fran
+    Press Key N Times    3    ${ARROW_DOWN}
+    Press Key N Times    1    ${ESC}
+
+    Press Key N Times And Enter    1    ${ARROW_DOWN}
+    Read From Terminal Until    Admin Pass
```
Comment (macpijan) [inline lib/usb-hid-msc-lib.robot:16]: This is quite unfortunate choice, as no every platform must support this

File: dasharo-compatibility/usb-hid-and-msc-support.robot [hunk 3]
```diff
@@ -59,14 +64,13 @@ USB001.003 USB devices detected by OS (Windows 10)
...
-#    ...    according to their labels.
-#    Power On
-#    Enter Tianocore
+USB002.001 USB keyboard detected in FW
+    [Documentation]    Check whether the external USB keyboard is detected
+    ...    correctly by the firmware and all basic keys work
+    ...    according to their labels.
+    Power On
+    Enter Setup Menu Tianocore
+    Navigate Through Some Menus
```
Comment (macpijan) [inline dasharo-compatibility/usb-hid-and-msc-support.robot:73]: I'm not sure if we need such bizzare keyword. Is not entering main menu, and perhaps going into another one, like `Enter Device Manager Submenu`

File: lib/usb-hid-msc-lib.robot [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    Press Key N Times    1    ${ENTER}
+    Read From Terminal Until    Physical
+    Press Key N Times    2    ${ESC}
+
+    Press Key N Times And Enter    1    ${ARROW_DOWN}
+    Press Key N Times    4    ${ARROW_DOWN}
+    Press Key N Times    4    ${ARROW_UP}
+    Press Key N Times    1    ${ESC}
+
+Find The Boot USB
```
Comment (macpijan) [inline lib/usb-hid-msc-lib.robot:32]: This is exactly against what we need. 
We are adding keyword that works only for one particular case (where PiKVM) is connected, and what is more, there are already existing keywords offering this: https://github.com/Dasharo/open-source-firmware-validation/blob/main/keywords.robot#L2896

File: dasharo-compatibility/usb-hid-and-msc-support.robot [hunk 5]
```diff
@@ -10,29 +10,32 @@ Library             RequestsLibrary
 # stuff in all test cases
 Resource            ../sonoff-rest-api/sonoff-api.robot
 Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../pikvm-rest-api/pikvm_comm.robot
 Resource            ../variables.robot
 Resource            ../keywords.robot
 Resource            ../keys.robot
+Resource            ../lib/secure-boot-lib.robot
```
Comment (macpijan) [inline dasharo-compatibility/usb-hid-and-msc-support.robot:17]: @filipleple You can add the new libraries to keywords.robot as a resource. Thanks to that you will not need to add them to each test module.

Comment (macpijan) [reply dasharo-compatibility/usb-hid-and-msc-support.robot:17]: and BTW, secure boot library would rather not be required here, if included this way

File: lib/usb-hid-msc-lib.robot [hunk 6]
```diff
@@ -0,0 +1,7 @@
+*** Keywords ***
+Upload And Mount DTS Flash ISO
+    [Documentation]    Mounts a bootable ISO as flash USB. Currently
+    ...    only the Qubes OS ISO seems to work for the platform.
+    Upload Image    ${PIKVM_IP}
```
Comment (macpijan) [inline lib/usb-hid-msc-lib.robot:5]: @filipleple Please update pikvm submodule. The name of this keyword is currently: `Upload Image To PiKVM` in the main branch as far as I can see....

File: lib/usb-hid-msc-lib.robot (hunk 1 above)
Comment (filipleple) [reply lib/usb-hid-msc-lib.robot]: Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)

File: lib/usb-hid-msc-lib.robot (hunk 2 above)
Comment (filipleple) [reply lib/usb-hid-msc-lib.robot]: Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)

File: dasharo-compatibility/usb-hid-and-msc-support.robot (hunk 3 above)
Comment (filipleple) [reply dasharo-compatibility/usb-hid-and-msc-support.robot]: Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)

File: lib/usb-hid-msc-lib.robot (hunk 4 above)
Comment (filipleple) [reply lib/usb-hid-msc-lib.robot]: Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)

File: lib/usb-hid-msc-lib.robot (hunk 6 above)
Comment (filipleple) [reply lib/usb-hid-msc-lib.robot]: Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)

File: dasharo-compatibility/usb-hid-and-msc-support.robot (hunk 5 above)
Comment (filipleple) [reply dasharo-compatibility/usb-hid-and-msc-support.robot]: Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)

File: .gitmodules [hunk 7]
```diff
@@ -7,9 +7,6 @@
 [submodule "pikvm-rest-api"]
 	path = pikvm-rest-api
 	url = git@github.com:3mdeb/pikvm-rest-api.git
-[submodule "lib"]
-	path = lib
-	url = git@github.com:3mdeb/testing-stands.git
```
Comment (mkopec) [inline .gitmodules:12]: why are we removing this submodule? is it no longer needed?

Comment (macpijan) [reply .gitmodules:12]: Not used. There are a few more things to remove from this repo I have missed removing when moving from githlab to github.

**Review by mkopec — APPROVED** [review]

File: .gitmodules (hunk 7 above)
Comment (mkopec) [reply .gitmodules:12]: Thanks, I just didn't see it explained in the commit message.

**Pull request merged**